readme.md

```diff
@@ -21,4 +21,11 @@ A few words about yourself <br>
 - **Professional status:** Backend Developer
 - **Interests:** Web Development, Machine Learning, Micro Services and System Design
 - **Hobbies**: Reading, Volunteering, Travelling
-- **About myself:** Hello! I’m Emon Hossain. I develop web applications. My core skill is based on JavaScript and I love to do most the things using JavaScript. I am a passionate programmer, developer, and learner. I love to eat and coding. 
\ No newline at end of file
+- **About myself:** Hello! I’m Emon Hossain. I develop web applications. My core skill is based on JavaScript and I love to do most the things using JavaScript. I am a passionate programmer, developer, and learner. I love to eat and coding. 
+
+**Md Ahnaf Sadik Inan**
+
+- **Academic Status:** 2nd year, B.Sc Honours, Departement of Mathematics, University of Dhaka, Bangladesh.
+- **Interests:** Pure Mathematics, Web Development, CP, AI(Machine Learning, Deep Learning and Data Science), Game Development(Designing and coding)
+- **Hobbies:** Reading, Playing Guitar, Gaming, Browsing random topics (I don't go out too much!)
+- **About Myself:** Hi! I'm Md Ahnaf Sadik Inan. As you can see, I have a lot of interests but no professional experience. Mathematica peaked my interest into coding. But I think my base language is Python. For web development I can work with JavaScript. This year I got a bit good working with fortran and C. Actually I  think I can adopt into any language if needed and am always interested to learn new things. 
```

Comment:
At the description, just use your ডাকনাম।  :-)

Reply:
I think I did it. Was a bit confusing though!

Reply:
To clear the confusion, what I meant was instead of using `Hi! I'm Md Ahnaf Sadik Inan`, just switch to something like `Hi! I'm Inan`, so that everyone knows how to address you. : -)

Reply:
Oh! My bad! I'll be careful about that next time! Thank you for letting me know! Is it still possible to to switch? The full request seems to closed and I did not make a branch! I'll be careful about them in the future too!

Reply:
No worries man. Not important for this particular repo. :-D